components/organisms/UserSettingsPage/user-settings-page.tsx

```diff
@@ -45,42 +100,61 @@ const UserSettingsPage = () => {
               <textarea
                 rows={4}
                 placeholder="Tell us about yourself."
-                className="bg-light-slate-4 rounded-lg px-3 py-2 "
+                className="bg-light-slate-4 rounded-lg px-3 py-2 disabled:cursor-not-allowed "
+                readOnly
+                value={
+                  userInfo?.bio ||
+                  "I am an open source developer with a passion for music and video games. I strive to improve the open source community and am always looking for new ways to contribute."
+                }
               ></textarea>
             </div>
             <TextInput
               classNames="bg-light-slate-4 text-light-slate-11 font-medium"
               placeholder="https://turtlepower.pizza"
               label="URL"
+              disabled
             />
             <TextInput
               classNames="bg-light-slate-4 text-light-slate-11"
               placeholder="@aprilcodes"
               label="Twitter Username"
+              disabled
+              value={`@${(userInfo && userInfo.twitter_username) || "open-sauced-user"}`}
             />
             <TextInput
               classNames="bg-light-slate-4 text-light-slate-11 font-medium"
               placeholder="StockGen"
               label="Company"
+              disabled
+              value={userInfo?.company || "open sauced"}
             />
             <TextInput
               classNames="bg-light-slate-4 text-light-slate-11 font-medium"
               placeholder="USA"
               label="Location"
+              disabled
+              value={userInfo?.location || "Romania"}
```

Comment:
```suggestion
              value={userInfo?.location || "Canada"}
```
I would love to like to leverage a country with a higher chance of usage.